code/game/gamemodes/cult/runes.dm

```diff
@@ -834,7 +834,7 @@ var/list/sacrificed = list()
 		if (cultist == user) //just to be sure.
 			return
 		if(cultist.buckled || cultist.handcuffed || (!isturf(cultist.loc) && !istype(cultist.loc, /obj/structure/closet)))
-			user << "<span class='warning'>You cannot summon \the [cultist], for \his shackles of blood are strong.</span>"
+			user << "<span class='warning'>You cannot call \the [cultist], for \he is bound by shackles of law. You must first break them! </span>"

```

Comment:
you can be cuffed by things other than handcuffs. or buckled. "By shackles of law" doesn't quite fit here.

Reply:
shackles of blood was also nonsensical and overly edgy, but I will muse on it.

Reply:
the entire gamemode is nonsensical and overly edgy. it's a blood cult

Reply:
Then the cult can be edgy all they like, but this is referencing someone being buckled to a chair or handcuffed. There's nothing supernatural about it.

Reply:
It also fails to adequately convey what the message means, for that you need to use another rune first to free them.